CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased](https://github.com/logiek/http-status-codes/compare/1.1.2...master)
 
+### Added
+- Added function to retrieve all status codes.
```

Comment:
Please specify that you are adding a function that retrieves all status codes with their corresponding reason phrase.

Reply:
✔️